Assets/MixedRealityToolkit.Services/InputSystem/GazeProvider.cs

```diff
@@ -273,7 +270,7 @@ public override void OnPostSceneQuery()
 
                 if (isDown)
                 {
-                    MixedRealityToolkit.InputSystem.RaisePointerDragged(this, MixedRealityInputAction.None, currentHandedness, currentInputSource);
```

Comment:
>InputSystem [](start = 40, length = 11)

Any idea why this didn't have a null check before? It looks like this system in Start() waits for input to become valid, so I'm curious how we need to keep checking that it's valid later.

Reply:
Good question. I have a habit of checking for null.... i can revert.

Reply:
removed